frontend/src/components/ViewTrips/trips-container.js

```diff
@@ -30,14 +30,18 @@ function queryUserTrips(db) {
  *
  * @param {Promise<!firebase.firestore.QuerySnapshot>} querySnapshot Promise
  *    object containing the query results with zero or more Trip documents.
+ * @param {EventHandler} handleEditTrip Displays the edit trip modal.
  * @return {Promise<!Array<Trip>>} Promise object containing an array
  *    of Trip React/HTML elements corresponding to the Trip documents included
  *    in `querySnapshot`.
  */
-function serveTrips(querySnapshot) {
+function serveTrips(querySnapshot, handleEditTrip) {
   return new Promise(function(resolve) {
     const tripsContainer = querySnapshot.docs.map(doc =>
-        ( <Trip key={doc.id} tripObj={doc.data()} tripId={doc.id} /> ));
+        ( <Trip tripData={doc.data()} tripId={doc.id}
+                handleEditTrip={handleEditTrip} key={doc.id} />
+        )
```

Comment:
I think it's good to have each element spaced out in this way when there's too many:
```suggestion
        ( 
            <Trip
                tripData={doc.data()}
                tripId={doc.id}
                handleEditTrip={handleEditTrip}
                key={doc.id}
            />
        )
```

Reply:
Yeah this does look cleaner. Fixed.